### DISCUSSION TITLE
超时

### DISCUSSION THREAD

**Discussion by sweanner** [monsterxcn/nonebot-plugin-gspanel#39] [提问]
![7139d21f04e9a73](https://user-images.githubusercontent.com/109018361/208033133-ef1968eb-a92d-429d-a677-8f64e3b8edfb.png)

**Comment by monsterxcn** [comment]
不清楚，是完全不能用吗？还是偶尔这样

**Comment by sweanner** [comment]
> 不清楚，是完全不能用吗？还是偶尔这样

不能用，一生成图片就报错

**Comment by monsterxcn** [comment]
什么系统？参考 [README - 使用须知](https://github.com/monsterxcn/nonebot-plugin-gspanel#%E4%BD%BF%E7%94%A8%E9%A1%BB%E7%9F%A5) 最后一条安装一下依赖然后再 `python -m playwright install` 试一试

**Comment by monsterxcn** [comment]
想到一个问题，你查询的角色是谁？

你能看看 `data/gspanel` 对应角色名文件夹下面的所有图片（命座、天赋、头像、抽卡图等）都成功下载了吗？还有 `weapon` `artifact` 文件夹下面该角色装备的物品图片都成功下载了吗？

**Comment by 173322993** [comment]
我也这样playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.然后这个是可以的吗
![Screenshot_20221221_182851_com termux](https://user-images.githubusercontent.com/111891236/208883613-a9456258-f5f5-4669-bb59-1803f73a8c10.jpg)

**Comment by monsterxcn** [comment]
@173322993 失败了，是浏览器需要的一些系统层面依赖无法安装，这个跟你的系统有关系，你是 Bot 跑在手机上吗？这个 Temux 还是啥我没用过，这装的是啥系统？
